topcoffea/modules/createJSON.py

```diff
@@ -60,6 +62,8 @@ def main():
     sampdic['treeName']     = treeName
     sampdic['histAxisName'] = histAxisName
     sampdic['options']      = options
+    if era is not None:
```

Comment:
I agree that sounds like a good check to include.

Reply:
@kmohrman do you have a preference on a warning vs an exception? I see @MatthewDittrich added a print statement for a warning. If I remember correctly, this script prints out a bunch of stuff, so I'm worried the warning might get buried.

Reply:
@bryates I think for me, it seems a warning is ok. It is conceivable that someone might want to make a R3 json that does not need the era listed; the era is only needed for overlap removal. So if e.g. you were just making one json for one dataset (for some test) you would not need the era included since you would not be doing any overlap removal.

Reply:
Ok good point. I wasn't sure if the era was necessary or just an option. This looks good then.